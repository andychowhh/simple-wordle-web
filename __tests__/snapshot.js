import { render } from "@testing-library/react";
import Home from "../pages/index";

it("renders homepage unchanged", () => {
  const home = render(<Home />);
  expect(home).toMatchSnapshot();
});
