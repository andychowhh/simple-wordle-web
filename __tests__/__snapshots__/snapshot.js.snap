// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="input__group__container"
      style="display: flex;"
    >
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Please enter verification code. Character 1"
          autocomplete="off"
          class="input__group__item  "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 2"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 3"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 4"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 5"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="input__group__container"
      style="display: flex;"
    >
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Please enter verification code. Character 1"
          autocomplete="off"
          class="input__group__item  "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 2"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 3"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 4"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 5"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="input__group__container"
      style="display: flex;"
    >
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Please enter verification code. Character 1"
          autocomplete="off"
          class="input__group__item  "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 2"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 3"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 4"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 5"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="input__group__container"
      style="display: flex;"
    >
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Please enter verification code. Character 1"
          autocomplete="off"
          class="input__group__item  "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 2"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 3"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 4"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 5"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="input__group__container"
      style="display: flex;"
    >
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Please enter verification code. Character 1"
          autocomplete="off"
          class="input__group__item  "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 2"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 3"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 4"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
      <div
        style="display: flex; align-items: center;"
      >
        <input
          aria-label="Character 5"
          autocomplete="off"
          class="input__group__item "
          maxlength="1"
          style="width: 1em; text-align: center;"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
